cirq-core/cirq/neutral_atoms/neutral_atom_devices_test.py

```diff
@@ -137,9 +137,11 @@ def with_qubits(self, new_qubits):
     with pytest.raises(ValueError, match="are too far away"):
         d.validate_operation(cirq.CZ.on(cirq.GridQubit(0, 0), cirq.GridQubit(2, 2)))
     with pytest.raises(ValueError, match="Too many Z gates in parallel"):
-        d.validate_operation(cirq.ParallelGateOperation(cirq.Z, d.qubits))
+        d.validate_operation(cirq.ParallelGate(cirq.Z, len(d.qubits)).on(*d.qubits))
     with pytest.raises(ValueError, match="Bad number of XY gates in parallel"):
-        d.validate_operation(cirq.ParallelGateOperation(cirq.X, d.qubit_list()[1:]))
+        d.validate_operation(
+            cirq.ParallelGate(cirq.X, len(d.qubit_list()[1:])).on(*d.qubit_list()[1:])
```

Comment:
It's a bit awkward to always have to specify the length and then separately call `on` to apply to qubits, especially as here when it's not a completely trivial expression to get the qubit list. Might be nice to add a helper function, similar to `cirq.measure` for `cirq.wait` for measure and wait gates, which have a similar problem in that they can be applied to any number of qubits. For example, could have a `cirq.parallel_gate` function:
```python
def parallel_gate(gate: 'cirq.Gate', *targets: 'cirq.Qid') -> 'cirq.Operation':
    return cirq.ParallelGate(gate, len(targets)).on(*targets)
```
The name is a bit of a misnomer since it actually returns an operation, not a gate, but it's the best I could come up with; `cirq.parallel` definitely seems to generic.

Reply:
Maybe I'll use `cirq.parallel_gate_operation(<gate>, <qubits>)` ?

Reply:
done.